docs/administrative-requirements.md

```diff
@@ -23,66 +23,61 @@ Preparing to connect your system to Enterprise Event Bus may require additional
+### OBI Access Form
 
-All VA systems are given an entry with a unique number in the eMASS system. The Enterprise Event Bus eMASS ID is 2350. 
+Event Bus Consumers consuming a BIP-sourced event that is available on the Event Bus must follow the OBI data access form process (aka the Corp DB Access form).
 
-Getting an eMASS ID is a prerequisite for System Categorization. 
+If the same consumer wants to consume additional BIP-sourced events at a later time, another OBI data access request form must be submitted to cover the use of that additional data.
 
-**For non-LHDI clients**:
+[Instructions for filling out the form (must be part of VA GitHub organization to view)](https://github.com/department-of-veterans-affairs/VES/blob/master/research/Event%20Bus/Partner%20Teams/BIP%20Meetings%20and%20Materials/SOP_Corporate_DB_Access_Approval%20081123%201.pdf) and [an example (must be part of VA GitHub organization to view)](https://github.com/department-of-veterans-affairs/VES/blob/master/research/Event%20Bus/Partner%20Teams/BIP%20Meetings%20and%20Materials/CorpDB%20Application%20Access%20Enterprise%20Event%20Bus%20full%20production.pdf) are available in the Enterprise Event Bus GitHub repository. 
 
-* If a prospective client system does not have an eMASS ID, it will need to submit a LEAF intake (formerly GRC intake) request: [Unified System Registry Intake (must be on VA network to view)](https://leaf.va.gov/NATIONAL/103/cybersecurity_request_portal/) to get one.
+Approval typically takes place in two stages: first for use in non-production environments, then for use in production.
 
-**For new LHDI clients**:
+Production approval will require that proof of ATO has been registered in eMASS.
 
-* For new teams on Lighthouse Delivery Infrastructure (LHDI), please consult your LHDI Enablement Liaison to start the LEAF intake request process and LHDI-specific onboarding. 
+### Sensitivity Filtering
 
-## Authority to Operate (ATO)
+The implementation of Sensitivity Filtering is required when event data, or data spawned from a callback initiated by the data, will be displayed through a VA application user interface to a VA employee or contractor.
 
-ATO, or “Authority to Operate,” indicates that your system has been evaluated by VA and given permission to deploy code in production.
+[Documentation for the rationale behind sensitivity filtering (must be part of VA GitHub organization to view)](https://github.com/department-of-veterans-affairs/VES/blob/master/research/Event%20Bus/Partner%20Teams/BIP%20Meetings%20and%20Materials/Benefits%20Information%20Sensitivity%20Filtering.md) and [instructions for implementing Sensitivity Filtering (must be part of VA GitHub organization to view)](https://github.com/department-of-veterans-affairs/VES/blob/master/research/Event%20Bus/Partner%20Teams/BIP%20Meetings%20and%20Materials/Benefits%20Information%20Sensitivity%20Level%20Filtering%20Options.md) can be found in the Enterprise Event Bus GitHub repository. 
```

Comment:
"Documentation for the rationale behind sensitivity filtering" --> "What is sensitivity filtering?"

Reply:
Are you saying we need to define what sensitivity filtering is first?

Reply:
No, I am suggesting that text be changed to better describe the linked page content. That page is more of an explanation of what sensitivity filtering is and how it should work, rather than a rationale (the why) for doing it.

Reply:
I made this change. Only thing I'm wondering is if we should add that as a title (H1) to the document.

Reply:
Let's leave as is for now.